superx/tests/test_livesearch.py

```diff
@@ -0,0 +1,47 @@
+"""
+import html parser
+"""
+import pytest
+from bs4 import BeautifulSoup
+# pylint: disable=no-else-continue
+
+
+@pytest.mark.run(order=4)
+def test_livesearch(client):
+    """
+    tests livesearch functionality
+    """
+    item_input1 = {'input': "שימורים"}
+    item_input2 = {'input': "sdsdd"}
+    item_input3 = {'input': "     "}
+    item_input4 = {'input': ""}
+
+    do_it(client, item_input1)
+    do_it(client, item_input2)
+    do_it(client, item_input3)
+    do_it(client, item_input4)
+
+
+def do_it(client, item_input):
+    """
+    execute the tests
+    """
+    with client:
+        response = client.get('/livesearch', data=item_input, follow_redirects=True)
+        cart_html = BeautifulSoup(response.data, 'html.parser')
+
+        is_inside = True
+        item_list = cart_html.findAll('td', {'id': True})
+
+        if not item_list:
+            is_inside = False
+            assert is_inside is False
+        else:
+            for item in item_list:
+                if item_input.get('input') in item.text:
+                    continue
+                else:
```

Comment:
while you disabled the check "no-else-continue", it is actually telling you that because of the continue, there is no need for the else.
In other words, what you coded is the exact same thing as:
```
            for item in item_list:
                if item_input.get('input') in item.text:
                    continue
                is_inside = False
                break

```

Reply:
that makes sense, fixed